src/utils/omit.ts

```diff
@@ -1,6 +1,5 @@
 import * as Lodash from 'lodash';
-import { Omit } from '../types/omit';
 
-export function omit<T, K extends keyof T>(obj: T, keys: K|K[]): Omit<T, K> {
+export function omit<T extends object, K extends keyof T>(obj: T, keys: K|K[]): Omit<T, K> {
```

Comment:
To make Lodash happy.

Reply:
I'm not convinced that `object` is the correct type here 🤔  But we'll see :)